LICENSE-CODE

```diff
@@ -0,0 +1,23 @@
+The code examples in this repository are licensed under the MIT License.
+
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
I think this would need to be filled, is owner the Lumi User Support Team?

Reply:
Thanks for spotting that!